src/main/kotlin/no/nav/syfo/application/exception/ApiError.kt

```diff
@@ -49,6 +50,13 @@ sealed class ApiError(val status: HttpStatusCode, val type: ErrorType, open val
             "No active sykmelding sent to employer"
         )
 
+    data object NoActiveArbeidsforholdError :
+        ApiError(
+            HttpStatusCode.Forbidden,
```

Comment:
Kunne vi tenke å gi `HTTP Status Code: 422 Unprocessable Entity`? 
Blir det litt mer spesifikk at feilen feiler in logic og ikke i access request?

Reply:
Chat GPT, forslår noe liknende, 
{
  "status": 422,
  "error": "Unprocessable Entity",
  "message": "The request cannot be processed because the provided information does not meet our system's requirements.",
  "details": [
    {
      "field": "business_connection",
      "issue": "Missing or invalid business connection. Please connect to the relevant business to proceed."
    },
    {
      "field": "system_logic",
      "issue": "The current system logic does not allow this operation under the provided conditions."
    }
  ]
}

Reply:
Dersom det ikke finnes et registrert arbeidsforhold på for det orgnummeret man er autentisert for så har man ikke lov/tilgang til å handle på vegne av brukeren. Tenker derfor at forbidden er mest riktig. Det er mest en "sikkerhetsvalidering".

422 kunne vi kanskje brukt på validering av innholdet i selve planen, men tenker det er for sent nå som vi har lansert dette i prod for Catalyst One, og to andre er godt i gang med implementering.